man/systemd.network.xml

```diff
@@ -700,6 +700,17 @@
           <literal>[IPv6PrefixDelegation]</literal> and the <literal>[IPv6Prefix]</literal>
           sections for more configuration options.
           </para></listitem>
+	</varlistentry>
+        <varlistentry>
+          <term><varname>IPv6PDSubnetId=</varname></term>
+          <listitem><para>Configure a specific subnet id from a received prefix delegation to
+          the interface. Allowed values are values from the range 0 to 9223372036854775808
+          (0x8000000000000000, 63 bit) to select a subnet id (as defined in
+          <ulink url="https://tools.ietf.org/html/rfc4291">RFC 4291</ulink>, section 2.5.4)
+	  or -1 (the default) to select an abitrary prefix. This option is only
```

Comment:
please don't overload "-1" like this. Just use a special value "no" instead or "off" or "auto" or "arbitrary" or so?